jboss-negotiation-extras/src/main/java/org/jboss/security/negotiation/AdvancedLdapLoginModule.java

```diff
@@ -328,6 +328,10 @@ public boolean login() throws LoginException
 
       if (result instanceof LoginException)
       {
+         if (log.isDebugEnabled())
```

Comment:
Not going to hold up merging but FYI there is no concatenation here so no need for the if statement, as soon as the log.debug call is entered the isDebugEnabled check will be performed for a second time.